src/replication.c

```diff
@@ -1135,11 +1374,23 @@ read_io_resp(spec_t *spec, replica_t *replica, io_event_t *revent, mgmt_cmd_t *m
 					free(*resp_data);
 					break;
 
+				case ZVOL_OPCODE_PREPARE_FOR_REBUILD:
```

Comment:
Status of zvol_io_hdr (i.e., ZVOL_OP_STATUS_OK or ZVOL_OP_STATUS_FAILED etc) need to be taken into consideration.
Some other commands also need to handle this.

Reply:
We also need to make sure that len field in zvol_io_hdr should be 0 in case of status not being OK

Reply:
Yes sir. Will try to make it uniform for all cases, handle_snap_create_resp is little different which need to be discussed.

Reply:
ok sir..